internal/pkg/service/buffer/store/model/column/column.go

```diff
@@ -213,7 +215,27 @@ func (Header) CsvValue(importCtx ImportCtx) (string, error) {
 	return string(header), nil
 }
 
-func (Template) CsvValue(_ ImportCtx) (string, error) {
+func (t Template) CsvValue(importCtx ImportCtx) (string, error) {
+	if t.Language == TemplateLanguageJsonnet {
+		ctx := jsonnet.NewContext()
+		ctx.GlobalBinding("body", jsonnet.ValueToLiteral(importCtx.Body.ToMap()))
+		headers := make(map[string]any)
+		for k := range importCtx.Header {
+			headers[k] = importCtx.Header.Get(k)
```

Comment:
`http.Header` je `map[string][]string` (každá hlavička je by design pole hodnot), nevím úplně proč, to je asi nějaký Go specifikum 🙂 ale myslím že si můžeme dovolit najít ke každé hlavičce jen jednu hodnotu.(?)

Reply:
Nie je to Go specifiku, hlavicka totiz moze byt definovana viac krat, je to nejaky edge-case v HTTP specifikacii.
Teda aby si mohol v Go kode na to nejak reagovat, tak je to `[]slice`.

Riesis to spravne `Get gets the first value associated with the given key.`.

Reply:
jj. to jsem četl. Spíš se jen chtěl nahlas ujistit že je ok ignorovat případný další hodnoty. 👍